Documentation/cmdref/cilium-agent_hive.md

```diff
@@ -27,6 +27,7 @@ cilium-agent hive [flags]
       --bpf-lb-source-range-all-types                             Propagate loadbalancerSourceRanges to all corresponding service types
       --bpf-node-map-max uint32                                   Sets size of node bpf map which will be the max number of unique Node IPs in the cluster (default 16384)
       --bpf-policy-map-max int                                    Maximum number of entries in endpoint policy map (per endpoint) (default 16384)
+      --bpf-policy-map-pressure-metrics-threshold float           Sets minimal rate for triggering policy map pressure metrics (default 0.1)
```

Comment:
Nit: "rate" might be misleading here. In my mind, that implies some rate of change.  Maybe something like "Sets threshold at which map pressure metrics for a particular map will be emitted" or similar?